protected/components/OParl10Object.php

```diff
@@ -388,20 +388,33 @@ private static function paper($id) {
         $antrag = Antrag::model()->findByPk($id);
 
         $data = [
-            'id'               => OParl10Controller::getOparlObjectUrl('paper', $antrag->id),
-            'type'             => self::TYPE_PAPER,
-            'body'             => OParl10Controller::getOparlObjectUrl('body', ($antrag->ba_nr != null ? $antrag->ba_nr : 0)),
-            'name'             => $antrag->getName(),
-            'reference'        => $antrag->antrags_nr,
-            'paperType'        => $antrag->getTypName(),
-            'auxiliaryFile'    => [],
-            'underDirectionOf' => [OParl10Controller::getOparlObjectUrl('organization', $antrag->referat_id, 'referat')],
-            'keyword'          => [],
-            'web'              => SITE_BASE_URL . $antrag->getLink(),
-            'created'          => OParl10Controller::mysqlToOparlDateTime($antrag->created),
-            'modified'         => OParl10Controller::mysqlToOparlDateTime($antrag->modified),
+            'id'                     => OParl10Controller::getOparlObjectUrl('paper', $antrag->id),
+            'type'                   => self::TYPE_PAPER,
+            'body'                   => OParl10Controller::getOparlObjectUrl('body', ($antrag->ba_nr != null ? $antrag->ba_nr : 0)),
+            'name'                   => $antrag->getName(),
+            'reference'              => $antrag->antrags_nr,
+            'paperType'              => $antrag->getTypName(),
+            'auxiliaryFile'          => [],
+            'originatorPerson'       => [],
+            'originatorOrganization' => [],
+            'underDirectionOf'       => [OParl10Controller::getOparlObjectUrl('organization', $antrag->referat_id, 'referat')],
+            'keyword'                => [],
+            'web'                    => SITE_BASE_URL . $antrag->getLink(),
+            'created'                => OParl10Controller::mysqlToOparlDateTime($antrag->created),
+            'modified'               => OParl10Controller::mysqlToOparlDateTime($antrag->modified),
         ];
 
+        foreach ($antrag->antraegePersonen as $ap) {
+            if ($ap->typ == AntragPerson::$TYP_GESTELLT_VON) {
+                $organization = Gremium::model()->findByName($ap->person->name);
```

Comment:
Does a `findByName()` method exist in the model?

Reply:
No that might be rather `->findByAttributes(["name" => ...])`. Though sure you want to find a "Gremium" by a person's name? Which information (in the UI) do you actually want to retrieve?

Reply:
In the UI (e.g. [here](https://www.muenchen-transparent.de/antraege/6869100)) the field "Gestellt von" which seems to be rendered by

https://github.com/codeformunich/muenchen-transparent/blob/25e1a323d5c1b4756a7169ebf5f2ad0fdf5f123a/protected/views/antraege/anzeige.php#L167-L175

If you could fix the data model / parser, this would of course be preferable.